### DISCUSSION TITLE
How do I use wrangler dev to bind a worker to 0.0.0.0?

### DISCUSSION THREAD

**Discussion by skrik2** [cloudflare/workers-sdk#7556] [Q&A] [answered]
I'm developing on a remote server (192.168.3.150).

```
pnpm create cloudflare@latest my-first-worker
cd my-first-worker 
pnpx wrangler dev --port 3000 --host 0.0.0.0
```

GG when browsing web pages

<img width="526" alt="Untitled" src="https://github.com/user-attachments/assets/9fd2bf3a-6409-47c3-9829-639068fb8197" />

How can I fix it? 
Any suggestions would be greatly appreciated!

**Comment by theoephraim** [comment, marked as answer]
The wrangler flag is `--ip` not `--host` (which I agree is pretty unexpected...)  [see docs](https://developers.cloudflare.com/workers/wrangler/commands/#dev)

If not specifying anything doesn't work, try the IP `--ip 192.168.3.150`
